lib/OrdinaryDiffEqDifferentiation/src/derivative_utils.jl

```diff
@@ -57,7 +89,29 @@ function calc_tderivative(integrator, cache)
         tf = cache.tf
         tf.u = uprev
         tf.p = p
-        dT = derivative(tf, t, integrator)
+
+        autodiff_alg = alg_autodiff(alg)
+        autodiff_alg = if autodiff_alg isa AutoSparse
+            autodiff_alg = ADTypes.dense_ad(autodiff_alg)
```

Comment:
```suggestion
            ADTypes.dense_ad(autodiff_alg)
```

Reply:
or, this could just be
```suggestion
         if autodiff_alg isa AutoSparse
            autodiff_alg = ADTypes.dense_ad(autodiff_alg)
        end # no else
```

Reply:
simplified by ADTypes v1.14